CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Categories for each release: Added, Changed, Deprecated, Removed, Fixed, Securit
 ### Added 
 
 * AWScliv2 asset to Nextflow applets
+* `dx extract_assay germline --retrieve-genotype` supports germline assay datasets with reference and no-call genotypes
```

Comment:
This, I believe, should go under the `Unreleased` section.

Reply:
Moved, thanks!